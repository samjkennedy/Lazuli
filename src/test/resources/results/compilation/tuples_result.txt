3
Hello
Hello
1
true
Hello
Hello
1
true
2
1
10
