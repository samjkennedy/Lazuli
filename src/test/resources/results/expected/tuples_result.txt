Hello
Hello
1
true
World
false
10
1
2
3
true